source/iostreams.tex

```diff
@@ -6854,25 +6854,31 @@
 without regard to the value of \tcode{os.exceptions()} and
 without turning on \tcode{ios_base::badbit} in the error state of \tcode{os}.
 \end{itemize}
+
+\par % This paragraph is part of the \effects clause.
 After constructing a \tcode{sentry} object,
 the function initializes a variable with automatic storage duration via
 \begin{codeblock}
 string out = vformat(os.getloc(), fmt, args);
 \end{codeblock}
+\begin{itemize}
+\item
 If the function is \tcode{vprint_unicode} and
-\tcode{os} is a stream that refers to a terminal capable of displaying Unicode
+\tcode{os} is a stream that refers to a terminal that
+is only capable of displaying Unicode via a native Unicode API,
```

Comment:
Per discussion on the LWG reflectors, "only" is ambiguous here.
```suggestion
\tcode{os} is a stream that refers to a terminal that
is capable of displaying Unicode only via a native Unicode API,
```
("Get out with this change that's not in the ingress paper and come back after the motion is merged" is a perfectly valid response to this suggestion.)

Reply:
This change was requested and agreed to on the LWG reflector. I think it qualifies as an editorial improvement that clarifies the intended meaning.

Reply:
Line 7849 needs the same change.

Reply:
Thanks, done!